client/src/templates/Challenges/components/VideoPlayer.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import YouTube from 'react-youtube';
+import type { BilibiliIds, VideoLocaleIds } from '../../../redux/prop-types';
+
+import envData from '../../../../../config/env.json';
+
+// TODO: pull these types from all-langs
+const { clientLocale } = envData as {
+  clientLocale:
+    | 'english'
+    | 'chinese'
+    | 'chinese-traditional'
+    | 'espanol'
+    | 'italian'
+    | 'portuguese';
```

Comment:
We use clientLocale in a few places - should we build this as an exported type declaration?

Reply:
Sounds good, yeah.  After https://github.com/freeCodeCamp/freeCodeCamp/pull/42862 gets in?

Reply:
Sorry, I just saw this issue. Here is a translated [bilibili video](https://www.bilibili.com/video/BV1Fh411z7tr) you can use to test. @ojeytonwilliams

Reply:
Thanks Miya, I'll put that in before we merge!